__tests__/topics.test.js

```diff
@@ -151,3 +151,59 @@ describe("/api/articles", () => {
       });
   });
 });
+
+describe("/api/articles/:article_id/comments", () => {
+  test("GET: 200, responds with an object containing a key of comments and a value of an array of comments", () => {
+    return supertest(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        // console.log(body);
+        expect(Object.keys(body)[0]).toBe("comments");
+      });
+  });
+  test("should respond with an array of comments for the given article_id", () => {
+    return supertest(app)
+      .get("/api/articles/9/comments")
+      .expect(200)
+      .then(({ body }) => {
+        // console.log(body);
+        expect(body.comments).toHaveLength(2);
+        body.comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            votes: expect.any(Number),
+            created_at: expect.any(String),
+            author: expect.any(String),
+            body: expect.any(String),
+            article_id: expect.any(Number),
```

Comment:
as you've requested comments for article 9 - you;d expect them all to have an id of 9 - at the moment, you could get comments for any article id and it would still pass

`article_id: 9`